includes/UserAdd.php

```diff
@@ -183,11 +197,10 @@
 
 	if( ! $stmt->execute() )
 	{
-		//echo "Error : $sqlConn->errno : $sqlConn->error <br>";
 		echo "Error in creating count database for user, contact Admin.";
 		die();
 	}
-
+	
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->